doc/user/content/sql/create-cluster-replica.md

```diff
@@ -47,6 +47,11 @@ _replica_name_ | A name for this replica.
 
 {{% replica-options %}}
 
+{{< note >}}
+If you do not specify an availability zone, Materialize will automatically assign the availability zone with the least existing replicas for the associated cluster. To view all replica availability zones, use
+the [`select * from mz_cluster_replicas`](/docs/sql/system-catalog/mz_catalog/#mz_cluster_replicas) command.
```

Comment:
The link doesn't work. You need to remove `/docs` from the start. Then the lint checker will pass, I believe.

Reply:
I think there's an opportunity to suggest _why_ we do this. Suggestion based on a chat with @teskje:

> If you do not specify an availability zone, Materialize will automatically assign the availability zone with the least existing replicas for the associated cluster to increase the cluster's tolerance to availability zone failure.

If you think that's too compact, you could unpack it in another sentence.